msb/network/zmq/publisher.py

```diff
@@ -25,7 +26,7 @@ def connect(self):
             sys.exit(-1)
 
     def send(self, topic: bytes, data: dict):
-        data = self.packer.dumps(data)
+        data = self.packer(data)
```

Comment:
If `packer` is a function like `json.dumps` the name `packer` is misleading.

Reply:
good point, I'd prefer `pack` and `unpack` as names